src/elements/hx-search-control/index.spec.js

```diff
@@ -0,0 +1,97 @@
+import { fixture, expect, oneEvent } from '@open-wc/testing';
+
+/**
+ * <hx-search-control> component tests
+ *
+ * @type HXSearchControlElement
+ *
+ */
+describe('<hx-search-control> component tests', () => {
+    const template = '<hx-search-control>';
+    const mockup = `<hx-search-control>
```

Comment:
Please see if this works by moving the `<hx-search-control>` above the code block:

```html
const mockup = `
        <hx-search-control>
          <input
            id="demoSearch"
            type="search"
          />
          <button
            type="button"
            class="hxClear"
            aria-label="Clear search"
            hidden>
            <hx-icon type="times"></hx-icon>
          </button>
          <hx-search></hx-search>
          <label
            for="demoSearch">
            Username
          </label>
        </hx-search-control>`;
```

Reply:
Changes have been done, please review.